packages/url_launcher/url_launcher_web/example/pubspec.yaml

```diff
@@ -8,6 +8,13 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  url_launcher:
+    path: ../../url_launcher/
+dependency_overrides:
+  url_launcher_web:
+    path: ../
+  url_launcher_platform_interface:
+    path: ../../url_launcher_platform_interface/
```

Comment:
@ditman I'm not qualified to review this part of the PR. Could you take a look?

Reply:
This will need to be reverted so we can merge and publish the affected packages in the correct order.